rcl/CMakeLists.txt

```diff
@@ -28,7 +28,6 @@ set(${PROJECT_NAME}_sources
   src/rcl/time.c
   src/rcl/timer.c
 )
-
```

Comment:
Why remove the empty line? Seems to help to distinguish the following macro from inline code.